.circleci/modernize/custom_fixers/fix_struct_unicode.py

```diff
@@ -0,0 +1,27 @@
+from lib2to3.fixer_base import BaseFix
+from lib2to3.pgen2 import token
+from lib2to3.fixer_util import Name, Node, Leaf, LParen, RParen, Dot, Newline
+import libmodernize
+
+
+class FixStructUnicode(BaseFix):
+    """
+        This fixer searches for places where we calling struct.pack or struct.unpack,
```

Comment:
`where we calling` should be `where we are calling`

Reply:
Thanks. Done.